### ISSUE TITLE
Install Chromedriver via npm?

### ISSUE THREAD

**Issue by alisterscott** [Automattic/wp-e2e-tests#770]
This would make running locally and on CI consistent and version controlled.

See: https://github.com/woocommerce/wp-e2e-webdriver/issues/21

What do you think @hoverduck?

**Comment by bsessions85** [comment]
I have played around with this a little bit and it works fine. The only issue I have seen is when trying to run tests in a docker container with my local directories mapped to the container. The tests will fail, presumably because the linux docker container is trying to use chromedriver made for mac. Running npm uninstall chromedriver followed by npm install should fix it in that situation. I'd like to give it a shot on CircleCI, but first we'll need to get a different version of the wp-e2e-tests image that doesn't have Chromedriver installed. I tried it with a local version of the image and everything ran fine.
